### DISCUSSION TITLE
Langgraph HIL with reasoning engine API

### DISCUSSION THREAD

**Discussion by BalasubramanyamEvani** [GoogleCloudPlatform/generative-ai#1587] [General]
Hello, I was conducting a feasibility study and was wondering if it's possible to use the Reasoning Engine API to leverage the LangGraph workflow we have with a human in the loop (using interrupts). We have a node that explicitly requests human input at a particular step in the graph.

Any example or sample to implement this would be helpful.

Thanks!

**Comment by holtskinner** [comment]
Hi, this PR creates a notebook for this use case https://github.com/GoogleCloudPlatform/generative-ai/pull/1673
